xbmc/music/windows/GUIWindowMusicNav.cpp

```diff
@@ -287,6 +287,10 @@ bool CGUIWindowMusicNav::GetDirectory(const std::string &strDirectory, CFileItem
       OnRetrieveMusicInfo(items);
   }
 
+  //Navigating music files so default content to "songs" unless in sources folder.
+  //This content allows view type to include media info so that file tag data can be displayed
+  if (!URIUtils::IsSourcesPath(strDirectory))
```

Comment:
>...files node should have content files only. This is how it's handled in the video section.

But video files to not have tag information, so nothing is lost when all files and folders are shown the same. Music files have additional information about the music, and upto the changes of #8011 (and the deprecation of CGUIViewStateWindowMusicSongs) it was displayed. It is that functionality that this PR is attempting to restore. 

Music users simply don't agree that the files listing should show only basic file information and nothing more. Now maybe I have implemented things poorly and there is a better way to restore this facility but the functionality is needed.